examples/systemcalls/systemcalls.c

```diff
@@ -92,8 +170,69 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *   The rest of the behaviour is same as do_exec()
  *
 */
-
+    openlog("do_exec_redirect function", 0, LOG_USER);
```

Comment:
@vidhyapalaniappan think about how you could avoid duplicating logic between these two functions.